integration/commands_administration_test.go

```diff
@@ -210,7 +210,6 @@ func TestCommandsAdministrationGetParameter(t *testing.T) {
 		"GetParameter_Asterisk1": {
 			command: bson.D{{"getParameter", "*"}},
 			expected: map[string]any{
-				"acceptApiVersion2": false,
```

Comment:
Not sure what to do about those. Should I create an issue about `getParameter`? @AlekSi WDYT?

Reply:
As discussed on the call, I'll just remove that `acceptApiVersion2` field from `getParameter`.